cypher/cypher-docs/src/docs/dev/glossary.asciidoc

```diff
@@ -360,8 +363,10 @@ The following commands are only executable against the `system` database:
 | <<access-control-database-administration-tokens, CREATE NEW RELATIONSHIP TYPE>>                | Schema                | Determines whether a user is allowed to create new relationship types on a specific database.
 | <<access-control-dbms-administration-role-management, CREATE ROLE>>                 | DBMS                  | Determines whether the user can create new roles.
 | <<access-control-dbms-administration-user-management, CREATE USER>>                 | DBMS                  | Determines whether the user can create new users.
+| <<access-control-dbms-administration-alias-management, ALTER MANAGEMENT>>           | DBMS                  | Determines whether the user can create, delete, and modify aliases.
```

Comment:
This should be ALIAS MANAGEMENT rather than ALTER MANAGEMENT. 

Also, as in the additions page, SHOW ALIAS is missing, both in this description and as its own entry